packages/renderer/src/lib/kube/KubernetesDashboard.svelte

```diff
@@ -0,0 +1,90 @@
+
+let nodeCount = $derived($kubernetesCurrentContextNodes.length);
+let deploymentCount = $derived($kubernetesCurrentContextDeployments.length);
+let serviceCount = $derived($kubernetesCurrentContextServices.length);
+let ingressRouteCount = $derived($kubernetesCurrentContextIngresses.length + $kubernetesCurrentContextRoutes.length);
+let pvcCount = $derived($kubernetesCurrentContextPersistentVolumeClaims.length);
+let configMapSecretCount = $derived(
+  $kubernetesCurrentContextConfigMaps.length + $kubernetesCurrentContextSecrets.length,
+);
+
+function openKubernetesDocumentation() {
+  window.openExternal('https://podman-desktop.io/docs/kubernetes');
+}
+</script>
+
+<NavPage searchEnabled={false}  title="Dashboard">
+  <svelte:fragment slot="bottom-additional-actions">
+    <div class="flex grow justify-end">
+      <KubernetesCurrentContextConnectionBadge />
+    </div>
+  </svelte:fragment>
+
+  <div class="flex min-w-full h-full" slot="content">
+    {#if noContexts}
+        <KubernetesEmptyPage />
+    {:else}
+        <div class="flex flex-col space-y-4 min-w-full overflow-y-auto">
+            <div class="flex flex-col gap-4 p-5 pb-2">
+                <span>Here you can manage and interact with Kubernetes clusters with features like connecting to clusters, and
+                viewing workloads like deployments and services.</span>
+
+                <span>Get up and running by clicking one of the menu items!</span>
+
+                <Link class="place-self-start" on:click={openKubernetesDocumentation}>Kubernetes documentation</Link>
+            </div>
+
+            <div class="flex flex-col gap-4 bg-[var(--pd-content-card-bg)] grow p-5">
+                {#if currentContextName}
+                    <div class="text-xl pt-2">Metrics</div>
+                    <div class="grid grid-cols-4 gap-4">
+                        <KubernetesDashboardResourceCard type='Nodes' Icon={NodeIcon} count={nodeCount} link='/kubernetes/nodes'/>
+                        <KubernetesDashboardResourceCard type='Deployments' Icon={DeploymentIcon} count={deploymentCount} link='/kubernetes/deployments'/>
+                        <KubernetesDashboardResourceCard type='Services' Icon={ServiceIcon} count={serviceCount} link='/kubernetes/services'/>
+                        <KubernetesDashboardResourceCard type='Ingresses & Routes' Icon={IngressRouteIcon} count={ingressRouteCount} link='/kubernetes/ingressesRoutes'/>
+                        <KubernetesDashboardResourceCard type='Persistent Volume Claims' Icon={PvcIcon} count={pvcCount} link='/kubernetes/persistentvolumeclaims'/>
+                        <KubernetesDashboardResourceCard type='ConfigMaps & Secrets' Icon={ConfigMapSecretIcon} count={configMapSecretCount} link='/kubernetes/configmapsSecrets'/>
+                    </div>
+                {/if}
+
+                <div class="text-xl pt-2">Explore articles and blog posts</div>
```

Comment:
question: mockup = blogposts, here: blog posts

Reply:
According to this I am just more parent than Tumblogger 😆:
https://english.stackexchange.com/questions/302723/blogpost-vs-blog-post#:~:text=It's%20a%20question%20of%20you,blog%20post'%20with%20your%20parents.

I tried to find other references - it seems like both are correct but blog post is still more common used on sites like Wikipedia.